ansible/vars/topo_t0-116.yml

```diff
@@ -134,6 +134,8 @@ topology:
         - 31
       vm_offset: 3
   DUT:
+    autoneg_interfaces:
+      intfs: [13, 14, 15, 16, 17, 18, 19, 20]
```

Comment:
Since auto negotiation needs to be matched between DUT and fanout ports. This definition means that both DUT and fanout configuration need to be updated while deploying minigraph. An extra step needs to be added to after deploy minigraph to configure fanout ports accordingly.

Reply:
discussed offline, resolved

Reply:
why 8 interfaces? i thought we said half of them an/lt, and half of none an/lt?

Reply:
changed it to 4 ports